tests/e2e/specs/merchant/merchant-disputes-submit-winning.spec.js

```diff
@@ -107,6 +107,14 @@ describe( 'Disputes > Submit winning dispute', () => {
 			text: 'Additional details',
 		} );
 
+		// Fill required additional text in order to make a winning dispute
+		// Used $eval as a workaround since .toFill won't work for this textarea
+		await page.click( '#inspector-textarea-control-3' );
```

Comment:
Would it be complicated to select the textarea with "Additional details" label (or inside container with that header), rather than rely on this sort of incidental selector?

Reply:
No, I added a better way which is about clicking the "Additional details" and filling the text into the selected field (clicked field). Now it is more specific.

Reply:
Unfortunately ended up with flakiness and very unstable test if I do with the above method. I left it with the first solution of selecting exact id of the field. The best solution would be to add additional test-automation-id which will be just for automation.

Reply:
Personally, I'm pretty uneasy about the current selector (which looks like it actually changes based on which product type is selected higher up in the page), but would totally support using an automation ID. Will ask others if there's any objection.

Reply:
Some further discussion on that culminated in choice to leave the selection by ID alone – is that correct?

These "inspector" IDs _appear_ to be from `@wordpress/components`, the original context for those components being the inspector in the block editor. Though that naming seems pretty contingent given the broader use of these components, I suppose they're unlikely to change at this point – I'd been worried that it was coming from somewhere other than the component source, like some higher-level library (though I'm still not sure precisely where the sequential IDs are being applied).

Reply:
Sorry, I don't understand this part `to leave the selection by ID alone`. The last time we spoke about this on Slack was to consider placing _:last-child_ for this textarea. Since the ID is unique for this textarea, `#inspector-textarea-control-3`, can we go as-is since no better selector available?

Reply:
Yeah, it'll do 😄 – thanks for exploring a set of other options though.

Reply:
Thanks :) Please consider approving this PR so I can merge it.

Reply:
I was good with the code changes but the E2E setup wasn't (and still isn't) working for me all of the sudden, and I didn't feel comfortable approving without that – @diegocurbelo lent a hand and ran them though 👍